sky_sync/open_weather_api/comm.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime
+import os
+
+import requests
+from requests import Response
+from requests.exceptions import ConnectionError, RequestException
+from rest_framework.exceptions import ParseError
+
+from .api_urls import (
+    build_current_weather_url,
+    build_direct_geocoding_url,
+    build_forecast_weather_url,
+    build_ip_url,
+    build_reverse_geocoding_url,
+)
+
+
+API_TOKEN = os.environ["OPEN_WEATHER_API_KEY"]
```

Comment:
lepiej to do jakichś settings.py przenieść

Reply:
Done, ta linijka jest w `settings.py`, a następnie zmienna jest importowana w `comm.py` z `settings.py`